targets/XPlatCppSdk/source/code/source/playfab/PlayFabPluginManager.cpp

```diff
@@ -99,4 +99,23 @@ namespace PlayFab
         return std::make_shared<OneDSCurlHttpPlugin>();
 #endif // PLAYFAB_PLATFORM_XBOX
     }
+
+    void PlayFabPluginManager::HandleException(const std::exception& ex)
```

Comment:
This is supposed to be implemneted by a user right ? Then don't pass the exception by reference